src/pyudev/_util.py

```diff
@@ -60,11 +60,10 @@ def ensure_unicode_string(value):
     Return the given ``value`` as unicode string.
 
     If the given ``value`` is not a unicode string, but a byte string, it is
-    decoded with the filesystem encoding (as in
-    :func:`sys.getfilesystemencoding()`).
+    decoded with utf-8.
```

Comment:
Thanks for the PR! How did you encounter this problem, and why don't you think it is necessary to change ```ensure_byte_string``` in a similar way?